config/locales/en.yml

```diff
@@ -1,32 +1,65 @@
-# Files in the config/locales directory are used for internationalization
-# and are automatically loaded by Rails. If you want to use locales other
-# than English, add the necessary files in this directory.
-#
-# To use the locales, use `I18n.t`:
-#
-#     I18n.t 'hello'
-#
-# In views, this is aliased to just `t`:
-#
-#     <%= t('hello') %>
-#
-# To use a different locale, set it with `I18n.locale`:
-#
-#     I18n.locale = :es
-#
-# This would use the information in config/locales/es.yml.
-#
-# The following keys must be escaped otherwise they will not be retrieved by
-# the default I18n backend:
-#
-# true, false, on, off, yes, no
-#
-# Instead, surround them with single quotes.
-#
-# en:
-#   'true': 'foo'
-#
-# To learn more, please read the Rails Internationalization guide
-# available at https://guides.rubyonrails.org/i18n.html.
-
 en:
+  activerecord:
+    attributes:
+      melange:
+        name: Name
+        event_date: Event date
+        participant_list: Participant list
+      participant:
+        role_list:
+          organizer: Organizer
+          moderator: Moderator
+          guest: Guest
+    model:
+      melange:
+        one: Event
+        few: Events
+        many: Events
+        other: Events
+        that_you_participate:
+          one: that you participate in.
+          few: that you participate in.
+          many: that you participate in.
+          other: that you participate in.
```

Comment:
If period is not connected with particular language maybe it is better to use it in the moment of constructing sentence.